pinot-spi/src/main/java/org/apache/pinot/spi/config/table/FieldConfig.java

```diff
@@ -67,7 +67,7 @@ public FieldConfig(@JsonProperty(value = "name", required = true) String name,
 
   // If null, there won't be any index
   public enum IndexType {
-    INVERTED, SORTED, TEXT, FST
+    INVERTED, SORTED, TEXT, FST, H3
```

Comment:
High level question, is it possible to have more than one of these index types on the same column?